backend/src/main/java/io/github/bbortt/event/planner/backend/web/rest/EventResource.java

```diff
@@ -103,7 +103,7 @@ public EventResource(EventService eventService, LocationMapper locationMapper, S
      */
     @PutMapping("/events")
     @PreAuthorize(
-        "@eventService.hasAccessToEvent(#event, \"" +
+        "@eventService.hasAccessToEvent(#event.id, \"" +
```

Comment:
this was because of a `NullPointerException` when checking access on event updates. but, actually this does not make any sense. double-check!